samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -174,11 +179,29 @@ static void lc3_decoder_thread(void *arg1, void *arg2, void *arg3)
 	}
 }
 
+static size_t get_chan_cnt(enum bt_audio_location chan_allocation)
+{
+	size_t cnt = 0U;
+
+	if (chan_allocation == BT_AUDIO_LOCATION_MONO_AUDIO) {
+		return 1;
+	}
+
+	while (chan_allocation != 0) {
+		cnt += chan_allocation & 1U;
+		chan_allocation >>= 1;
+	}
```

Comment:
```suggestion
	while (chan_allocation != 0) {
		chan_allocation &= (chan_allocation - 1U);
		cnt++;
	}
```

Reply:
Can you elaborate on why?

Reply:
It only iterate for the bits that are set.

Reply:
Ah, that makes sense. 

I would argue that it's a minor optimization (this function is rarely called), that is less readable and will effectively be replaced by https://github.com/zephyrproject-rtos/zephyr/issues/69617 in the future. 

Do you still request this change?

Reply:
> Do you still request this change?
No.